README.md

```diff
@@ -23,3 +23,17 @@ For more information about the OpenGitOps project, see: <https://github.com/open
 ## Code of Conduct
 
 The OpenGitOps community follows the [CNCF Code of Conduct](https://github.com/open-gitops/.github/blob/main/CODE_OF_CONDUCT.md).
+
+## Refence this Content
+
+A [BibTex](https://www.bibtex.org) code snippet to use as reference for the content available in this repository is listed below, make sure to change your site visit date accordinly.
+
+```
+@misc{open_gitops,
+  author       = {{Open GitOps Project}},
+  title        = {{Open GitOps Documents - Glossary and Principles}},
```

Comment:
@samueltauil I would call it "GitOps Principles and Glossary." It is maintained by, and lives in this repo under, the CNCF OpenGitOps project.

For authors list, see the "Attribution" section at https://github.com/open-gitops/documents/releases/tag/v1.0.0, It is also in the git commits, but listed here for easy access – alphabetized by last name (35 people). Question – would it be helpful for BibText to have an actual file listing authors somewhere in the content that you can point to?

Note that there will be more authors in future versions of the principles, but I assume we can update this BibTex snippet when that happens? Speaking of, should the version and it's release date be added to the `note` and `year` below?

Reply:
So usually the number of authors in citations falls into the default of 3 and then when rendered with more than that it appends _et al_. In order to list all those 35 authors it would be needed for the person writing that study to modify with the attibute `maxbibnames=35` in the `\usepackage` include command in the preambule in their primary tex file. If we decide to go ahead and add all 35 authors when researchers refer to the content in their bibliography, then we could externalize the list of authors in a separate .bib file and include the content of it in the snippet which would likely to force people to generate another bib file to their structure only to have our authors list properly displayed. My suggestions would be to refer the authors as a project like it is now and leave validation at the discretion of who is willing to validate it or we can create a .bib file listing all authors and use the `crossref` in the @misc entry to refer that external file (which would make easier for us to update/change). 

```
@person{author1,
    name = {Author One}
}

@person{author2,
    name = {Author Two}
} 
```

and then refer that like this: 

``` 
@misc{referencekey,
    title = {Title of the Misc Entry},
    year = {Year},
    crossref = {authors}
}
```

I would rather prefer to refer the list of authors to be validated at our repo instead of forcing a more complicated approach where we would eventually have to force people to add another file to their research content only to proper display our citation list of authors.

In relation to the title attribute I'll change it to what you suggested `GitOps Principles and Glossary` 👍

Reply:
I agree this makes the most sense 👍